### ISSUE TITLE
Different coordinates

### ISSUE THREAD

**Issue by pbelmans** [superficie/superficie-algebriche#4]
Right now the chart is implemented for c_2 and c_1^2. But it seems that the pair c_1^2 and chi is also in use (see http://arxiv.org/pdf/math/0602477.pdf).

So one should in theory be allowed to choose two independent numerical invariants, and then the map should be drawn for these invariants.

**Comment by pbelmans** [comment]
Now that we have all numerical invariants available in a surface, it is not too hard to implement different coordinate systems. I.e. it would become possible to implement a chart with c_1^2 and \chi!

**Comment by jcommelin** [comment]
Yes, I hoped that this wouldn't be to hard now. I am not yet sure how to do the UX though.
How will we let people choose the axis? Using dropdown's?
And do you need to rewrite a lot of the chart-generating code to enable this, or is it going to be “easy-peasy-lemon-squeezy”?

**Comment by pbelmans** [comment]
It'll be mostly copy-paste, and I draw the nodes in different `svg` elements, and then I just show the correct `svg`. I should try to refactor some things to improve the code layout though...

Regarding user-interface: I'd add two (or more) links below the graph, specifying the coordinate system.

**Comment by jcommelin** [comment]
Ok, I guess computationally it isn't very heavy yet, so drawing multiple `svg`'s but showing only one is not too bad. Good idea.

Concerning UX: I think that is fine as well. [I'm always thinking too complicated (-; …] But I do think that all the {6 \choose 2} = 15 pairs of invariants provide potentially interesting charts. So I don't know if that won't be too much of a copy-pasting?

**Comment by pbelmans** [comment]
I only do (c_1^2,chi). If you can show me other visualisations in the literature, we can think about these too. But for (c_1^2,chi) and (c_2,c_1^2) the relevant bounds make sense, whereas that is not necessarily the case for an arbitrary choice.

**Comment by jcommelin** [comment]
Well, I'd really like to be able to use this tool to quickly answer questions like: What values can p_g have if chi = 1? So even if such visualisations are not in the literature, I think they are quite useful.

**Comment by pbelmans** [comment]
I'll see how easy the refactoring goes, I won't make promises. The problem is also that you need to choose a good range, to make sure interesting stuff is happening. That is my biggest concern actually. I don't want to implement scaling. It's already taking up enough time as it is...

**Comment by pbelmans** [comment]
I played around a bit with different coordinates, but nothing really struck me as amazing.
